pkg/sync/rpc/server.go

```diff
@@ -725,7 +725,7 @@ func (s *SyncAgentServer) BackupStatus(ctx context.Context, req *ptypes.BackupSt
 }
 
 func (*SyncAgentServer) BackupRemove(ctx context.Context, req *ptypes.BackupRemoveRequest) (*empty.Empty, error) {
-	cmd := reexec.Command("sbackup", "delete", req.Backup)
+	cmd := reexec.Command("backup", "delete", req.Backup)
```

Comment:
Is there any place using `BackupRemove`? I thought only longhorn-manager will do backup deletion but why sync agent in replica has this capability as well?

Reply:
No, `BackupRemove` is not used. longhorn-manager calls `backup rm...` rather than `BackupRemove` to delete a backup.

Reply:
ref: [/controller/backup_controller.go#L310-L314](https://github.com/longhorn/longhorn-manager/blob/3b1d6252cd8389e8465ee4b38cf93d6543f8c372/controller/backup_controller.go#L310-L314)
```go
			backupURL := backupstore.EncodeBackupURL(backup.Name, backupVolumeName, backupTargetClient.URL)
			if err := backupTargetClient.BackupDelete(backupURL, backupTargetClient.Credential); err != nil {
				log.WithError(err).Error("Error deleting remote backup")
				return err
			}
```

Reply:
Keeping in sync server seems being confused somehow if it's not used. However, this is an exported interface, so let's just keep it as is.